thunder/core/transforms.py

```diff
@@ -1443,6 +1443,20 @@ def _log_sigmoid_grad(
 register_grad("torch.nn.functional.logsigmoid", _log_sigmoid_grad)
 
 
+def _softplus_grad(a: TensorProxy, /, beta: float = 1.0, threshold: float = 20.0):
```

Comment:
Neat!

Did this actually improve performance?

Reply:
I didn't do rigorous profiling, but it seems like it helped a smidge.